### PR TITLE
[Edit Profile] the buttons of the 'Edit Picture' pop-up have wrong position after resizing #2637

### PR DESCRIPTION
**Pull request by Edw1x**
# Before: 
## All UI controls have wrong position after resizing
image
image
image

# Result needed:
## All UI controls are visible on the screen during and after resizing
image
image

# After:
image
image

# Additional bug fixed:

### CODE REVIEW

**Review by JimHolman1199 — APPROVED** [review]

**Comment by JimHolman1199** [comment]
Congratulations on the first PR, well done! &#127829;

**Review by idzhersh — APPROVED** [review]

**Review by yuriiLisunov — APPROVED** [review]

File: src/app/component/shared/components/edit-photo-pop-up/edit-photo-pop-up.component.scss [hunk 1]
```diff
@@ -49,44 +48,34 @@ $secondErrorColor : #b42626;
...
-        font-weight: 600;
+        font-weight: bold;
         font-size: 16px;
-        height: 48px;
-        line-height: 16px;
+        line-height: 24px;
       }
 
       .primary-global-button {
         background: #fff;
```
Comment (VitalyBorysenko) [inline src/app/component/shared/components/edit-photo-pop-up/edit-photo-pop-up.component.scss:66]: maybe better to create a variable for this color (such as --primary-white)?

File: src/app/component/shared/components/edit-photo-pop-up/edit-photo-pop-up.component.scss [hunk 2]
```diff
@@ -95,17 +84,15 @@ $secondErrorColor : #b42626;
...
-          border: 1px solid var(--primary-grey);
+          color: var(--primary-green);
+          background: transparent;
+          border: 1px solid var(--primary-green);
         }
       }
 
       .secondary-global-button {
         background: var(--primary-green);
         color: #fff;
```
Comment (VitalyBorysenko) [inline src/app/component/shared/components/edit-photo-pop-up/edit-photo-pop-up.component.scss:95]: same propose.

File: src/app/component/shared/components/edit-photo-pop-up/edit-photo-pop-up.component.scss (hunk 1 above)
Comment (Edw1x) [reply src/app/component/shared/components/edit-photo-pop-up/edit-photo-pop-up.component.scss]: Added

File: src/app/component/shared/components/edit-photo-pop-up/edit-photo-pop-up.component.scss (hunk 2 above)
Comment (Edw1x) [reply src/app/component/shared/components/edit-photo-pop-up/edit-photo-pop-up.component.scss]: Added

**Review by VitalyBorysenko — APPROVED** [review]

**Review by SofiiaBaziv — APPROVED** [review]

**Review by mr-pavlenko — APPROVED** [review]

**Pull request merged**